package.json

```diff
@@ -18,7 +18,7 @@
     "url": "git@github.com:Kinvey/nativescript-sdk"
   },
   "scripts": {
-    "build": "del 'src/**/*.js' && del 'push/**/*.js' && tsc",
+    "build": "rm -rf 'src/**/*.js' && rm -rf 'push/**/*.js' && tsc",
```

Comment:
I actually don't think this is necessary since TS compiler just replaces these files when it emits new ones. Even with the rm -rf it depends on the "terminal" being used and what it supports, but that's always going to be the case, but I think this is simple and popular enough and we needn't add specific libraries for cross platform deletion of files :)